CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.1.6
```

Comment:
I believe you should Add an new version instead of removing this one, no?

Reply:
I don't know what you mean, I did add a new version? 


If you are talking about v1 embedding it has been deprecated for.a  while Now, it should be removed anyway

Reply:
I think I know what he mean, you didn't add specific new version on this changelog like the 8.1.5 one. So better you add it as 8.1.6 and the package owner should merge this pr then publish to pub.dev

Reply:
I'm not a maintainer but the changelog looks just fine to me.

Reply:
I've tested this on my setup, and everything works as expected. Great job! @miguelpruivo

Reply:
> I don't know what you mean, I did add a new version?

Sorry, my bad... I did not have my coffee yet when I reviewed :)

Reply:
Bumping version is good until another PR with version bump is merged. It is up to the maintainer if he requires version bump as part of the PR. Personally I do the version bump in the release branch dedicated the version release. I know that for example Flutter community plugins require version bump (I've been through it and got hindered several times due to other PRs with version merging and having to resolve conflict and rebase).
Just merge this PR (🙏🏻 🙏🏻 🙏🏻) with or without version FFS.